### DISCUSSION TITLE
How do I alter the colours of the status bar?

### DISCUSSION THREAD

**Discussion by ghost** [NvChad/NvChad#1184] [Q&A] [answered]
I'm attempting to write a custom theme. How and where do I define the colours for the status (Feline?) bar? Specifically the background light grey tone; I think I've managed to theme the rest.

![SS](https://user-images.githubusercontent.com/102994494/172465810-d25c6d42-9b17-497b-b34f-d267991ee922.png)

**Comment by budchirp** [comment, marked as answer]
Nvchad isn't using feline, it uses their own statusline for 2 days.

https://github.com/NvChad/base46/blob/master/lua/integrations/statusline.lua

**Reply by ghost** [reply]
Good to know. I wasn't sure what it was called and the ui-related-plugins section still labels it as Feline.

**Reply by budchirp** [reply]
Probably @siduck didn't update the readme

**Comment by siduck** [comment]
@CanKolay3499 the user can still override the default statusline highlights,  we had defined highlight groups in feline too so there shouldn't be much difference, I'll update the readme
